src/rt/cover.d

```diff
@@ -149,10 +149,83 @@ uint parseNum(const(char)[] s)
 T min(T)(T a, T b) { return a < b ? a : b; }
 T max(T)(T a, T b) { return b < a ? a : b; }
 
+version (unittest) __gshared bool inUnittest;
+
+string parseMergeOpt(string opt) @nogc nothrow
+{
+    switch (opt)
+    {
+        case "no", "No", "NO", "n", "N", "0":
+            merge = false;
+            return "0";
+        case "", "yes", "Yes", "YES", "y", "Y", "1":
+            merge = true;
+            return "1";
+        default:
+            if (!inUnittest)
+                fprintf(stderr, "DRT Error: Runtime option 'cov-merge' should "
+                        "be 1/y/yes or 0/n/no, got '%.*s'.\n",
```

Comment:
No implicit string concatenation please.

Reply:
Argh! I saw that and thought I had fixed it already.

Reply:
Updated.